test/bdd/features/search.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Searching for assets on the DKG
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+  Scenario: Publish and search assertion on the network with keywords
+    Given I setup 4 nodes
+    When I call publish on node 1 with validAssertion with keywords:
+      | keyword 1 | keyword 2 |
+    And I wait for last publish to finalize
+    And Last publish finished with status: COMPLETED
+    Given I call search request on node 1 with validAssertion for the keywords:
+      | keyword 1 | keyword 2 |
+    And I wait searching request to be finalized
```

Comment:
When defining the steps you should always have in mind that this step can be used in any other test so it should be more descriptive e.g.:
I wait searching request to be finalized -> I wait for last search request to finalize
I get the metadata which contains the keywords... -> Metadata from last search request contains keywords...